pyproject.toml

```diff
@@ -104,14 +108,16 @@ include = [
   "instructlab.generator",
   "instructlab.cli",
   "instructlab.cli.model",
-  "instructlab.model",
   "instructlab.cli.process",
+  "instructlab.cli.rag",
+  "instructlab.llamacpp",
```

Comment:
some changes are not related to your PR L113-115

Reply:
Just re-alphabetizing the list -- it has been alphabetical and then got out of order in a recent PR so I am restoring that.